semaphore/test.sh

```diff
@@ -2,3 +2,5 @@
 set -e
 
 mix test
+mix coveralls.json
+bash <(curl -s https://codecov.io/bash)
```

Comment:
You might want to add `|| exit 0` here if you don't want a failed coverage upload to fail the build.

Reply:
Thanks, that does indeed seem preferable.